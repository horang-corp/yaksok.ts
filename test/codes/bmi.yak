약속, 키 [키] 몸무게 [몸무게]의 비만도
    결과: 몸무게 / (키 / 100 * 키 / 100)

비만도: 키 [177] 몸무게 [68]의 비만도
비만도 보여주기
