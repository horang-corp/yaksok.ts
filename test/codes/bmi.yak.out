21.705129432793896
